### PR TITLE
Fix: nested plan for server manual

### PR DESCRIPTION
**Pull request by rickwu666666**
Fix: nested plan for server manual

### CODE REVIEW

**Review by stanley31huang — CHANGES_REQUESTED** [review]
there're two incorrect nested plan in the manual test plan for server, please also fix it as well.

**Review by zongminl — CHANGES_REQUESTED** [review]

File: checkbox-provider-ce-oem/units/test-plan-ce-oem-full-classic.pxu [hunk 1]
```diff
@@ -51,7 +51,7 @@ _description:
 estimated_duration: 3600
 include:
 nested_part:
-    ce-oem-automated
+    ce-oem-manual
     com.canonical.certification::client-cert-iot-server-20-04-manual
     after-suspend-ce-oem-automated
```
[the remark this answers is not in the document]
Comment (rickwu666666) [reply checkbox-provider-ce-oem/units/test-plan-ce-oem-full-classic.pxu]: Updated. Thanks!

**Comment by rickwu666666** [comment]
> there're two incorrect nested plan in the manual test plan for server, please also fix it as well.

Updated. Thanks!

**Review by LiaoU3 — APPROVED** [review]
LGTM

**Review by patliuu — APPROVED** [review]
LGTM

**Pull request merged**